### DISCUSSION TITLE
how can i download obs 27.2.4?

### DISCUSSION THREAD

**Discussion by caituoyu** [obsproject/obs-studio#7355] [General]
anyone can give me a DMG？thanks.
m1 pro 
After obs is updated to 28, reinstall and delete the registry, the screen recording function cannot be turned on, and the system shows that it has been ticked。

**Comment by Fenrirthviti** [comment]
This sounds more likely to be a support request, and we are not currently accepting support requests on GitHub Discussions. Make sure you read any [pinned posts](https://github.com/obsproject/obs-studio/discussions/5206) on discussion boards before opening a new discussion. Please use our [forums](https://obsproject.com/forum/) or [Discord](https://obsproject.com/discord/) for further assistance.

Thank you!
